Resources/Prototypes/Entities/Clothing/Ears/headsets.yml

```diff
@@ -5,46 +5,69 @@
   name: headset
   description: An updated, modular intercom that fits over the head. Takes encryption keys.
   components:
+  - type: ContainerContainer
+    containers:
+      key_slots: !type:Container
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyCommon
   - type: Headset
+    defaultChannel: Common
+    keysExtractionMethod: Screwing
   - type: Sprite
     state: icon
   - type: Clothing
     slots:
     - ears
     sprite: Clothing/Ears/Headsets/base.rsi
 
+- type: entity
+  parent: ClothingHeadset
+  id: ClothingHeadsetGrey
+  name: passenger headset
+  components:
+  - type: Sprite
+    sprite: Clothing/Ears/Headsets/base.rsi
+
 - type: entity
   parent: ClothingHeadset
   id: ClothingHeadsetCargo
   name: cargo headset
   description: A headset used by supply employees.
   components:
   - type: Headset
-    channels:
-    - Common
-    - Supply
     defaultChannel: Supply
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyCargo
   - type: Sprite
     sprite: Clothing/Ears/Headsets/cargo.rsi
   - type: Clothing
     sprite: Clothing/Ears/Headsets/cargo.rsi
 
 - type: entity
-  parent: ClothingHeadset
+  parent: ClothingHeadsetCargo
+  id: ClothingHeadsetMining
+  name: mining headset
+  description: Headset used by shaft miners.
+  components:
+  - type: Sprite
+    sprite: Clothing/Ears/Headsets/mining.rsi
+  - type: Clothing
+    sprite: Clothing/Ears/Headsets/mining.rsi
+
+- type: entity
+  parent: ClothingHeadsetCargo
   id: ClothingHeadsetQM
   name: qm headset
   description: A headset used by the quartermaster.
   components:
-  - type: Headset
-    channels:
-    - Command
-    - Common
-    - Supply
-    defaultChannel: Supply
-  - type: Sprite
-    sprite: Clothing/Ears/Headsets/cargo.rsi
-  - type: Clothing
-    sprite: Clothing/Ears/Headsets/cargo.rsi
```

Comment:
why was this deleted?

Reply:
Parenting

Reply:
![image](https://user-images.githubusercontent.com/46600554/212152684-afc5b688-5804-4eb9-a60f-4dd137ec63ae.png)